diff -Nur PyQt-mac-gpl-4.10/configure.py PyQt-mac-gpl-4.10.new/configure.py
--- PyQt-mac-gpl-4.10/configure.py	2013-03-01 22:57:30.000000000 +0100
+++ PyQt-mac-gpl-4.10.new/configure.py	2013-04-14 13:30:16.638876596 +0200
@@ -2112,11 +2112,11 @@
 
     out << QLibraryInfo::licensee() << '\\n';
 
-#if defined(QT_SHARED) || defined(QT_DLL)
+// #if defined(QT_SHARED) || defined(QT_DLL)
     out << "shared\\n";
-#else
-    out << "\\n";
-#endif
+// #else
+//    out << "\\n";
+// #endif
 
     // Determine which features should be disabled.
 
