diff -Nur PyQt-mac-gpl-4.10/pyrcc/rcc.cpp PyQt-mac-gpl-4.10.new/pyrcc/rcc.cpp
--- PyQt-mac-gpl-4.10/pyrcc/rcc.cpp	2013-03-01 22:57:58.000000000 +0100
+++ PyQt-mac-gpl-4.10.new/pyrcc/rcc.cpp	2013-04-14 13:28:53.349332217 +0200
@@ -407,8 +407,7 @@
     fprintf(out, "# -*- coding: utf-8 -*-\n\n");
     fprintf(out, "# Resource object code\n");
     fprintf(out, "#\n");
-    fprintf(out, "# Created: %s\n", QDateTime::currentDateTime().toString().toUtf8().constData());
-    fprintf(out, "#      by: The Resource Compiler for PyQt (Qt v%s)\n", QT_VERSION_STR);
+    fprintf(out, "# Created by: The Resource Compiler for PyQt (Qt v%s)\n", QT_VERSION_STR);
     fprintf(out, "#\n");
     fprintf(out, "# WARNING! All changes made in this file will be lost!\n");
 
